docs/connecting-to-a-lit-network/migrating-to-datil.md

```diff
@@ -5,29 +5,34 @@ import TabItem from '@theme/TabItem';
 
 Lit is launching three new networks, Datil-dev, Datil-test, and Datil, to improve performance and stability for production users compared to previous Lit networks like Cayenne, Manzano, and Habanero.
 
-The Datil networks use a new rollup blockchain called [Chronicle Vesuvius](./lit-blockchains/chronicle-vesuvius), which replaces the [Chronicle](./lit-blockchains/chronicle) blockchain that powered Lit's earlier networks. You will learn how to migrate data between these chains below.
+The Datil-dev and Datil-test networks use a new rollup blockchain called [Chronicle Yellowstone](./lit-blockchains/chronicle-yellowstone.md). These new blockchains replace the [Chronicle](./lit-blockchains/chronicle) blockchain that powered Lit's earlier networks. You will learn how to migrate data between these chains below.
 
 Depending on the existing Lit network you are using, you should migrate to its corresponding Datil network to take advantage of these improvements:
 
 | Currently Available | Requires Payment | Minimum Lit Package Version | Your Current Network | Network to Migrate to | Description                                                  |
 |---------------------|------------------|-----------------------------|----------------------|-----------------------|--------------------------------------------------------------|
-| ❌                   | ✅                | n/a                         | `habanero`           | `datil`               | Decentralized mainnet designed for production use cases      |
-| ❌                   | ✅                | n/a                         | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
-| ✅                   | ❌                | `6.1.0`                     | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
+| ❌                   | ✅                | n/a                         | `habanero`           | `datil-prod`               | Decentralized mainnet designed for production use cases      |
+| ✅                   | ✅                | `6.2.2`                     | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
+| ✅                   | ❌                | `6.2.2`                     | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
```

Comment:
Payment is required after 3 free requests per day?

Reply:
How are the number of requests tracked? Are they tied to IP address?